.github/workflows/test-only.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Build changedetection.io container for testing
         run: |         
           # Build a changedetection.io container and start testing inside
-          docker build . -t test-changedetectionio
+          docker build --build-arg LOGGER_LEVEL=TRACE -t test-changedetectionio .
```

Comment:
This set LOGGER_LEVEL env.
It affects the container image of this test and pytest `tests/conftest.py`